src/sasl_plain.c

```diff
@@ -66,8 +67,17 @@ CONCRETE_SASL_MECHANISM_HANDLE saslplain_create(void* config)
                 else
                 {
                     /* Ignore UTF8 for now */
-                    result->init_bytes = (unsigned char*)malloc(authzid_length + authcid_length + passwd_length + 2);
-                    if (result->init_bytes == NULL)
+                    //size_t malloc_size = authzid_length + authcid_length + passwd_length + 2);  // use safe add
```

Comment:
Remove?

Reply:
no, I want it there to show the expected math